doc/source/devcontainer.rst

```diff
@@ -0,0 +1,22 @@
+VS Code Remote - Containers 
+======================================
+
+When working on the Flower framework we want to ensure that all contributors use the same developer environment to format code or run tests. For this purpose we are using the VSCode Remote Containers extension. What is it? Read the following quote:
+
+`Quoting the offical VSCode documentation <https://code.visualstudio.com/docs/remote/containers>`_
```

Comment:
Was this line intended to show a quote?

Reply:
Yes. Thats the reason why the subsequent two lines are indented. That link is only the source of the quote.

Reply:
Change a bit so its not confusing